client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -199,6 +204,16 @@ const DataFilesCopyModal = React.memo(() => {
           </div>
         </div>
       </ModalBody>
+      <div>
+        {modalParams.scheme === 'public' && (
+          <ModalFooter className="d-flex justify-content-start">
+            <SectionMessage type="warning">
+              Files copied to Public Data will be avaliable to general public.{' '}
+              <b>This action cannot be reversed.</b>
+            </SectionMessage>
+          </ModalFooter>
+        )}
+      </div>
```

Comment:
I think you can actually drop this outer `<div>`
```suggestion
      {modalParams.scheme === 'public' && (
        <ModalFooter className="d-flex justify-content-start">
          <SectionMessage type="warning">
            Files copied to Public Data will be avaliable to general public.{' '}
            <b>This action cannot be reversed.</b>
          </SectionMessage>
        </ModalFooter>
      )}
```